### PR TITLE
feat: add allContributors to recognise contributors to the project

### PR DESCRIPTION
**Pull request by raizo07**
# Pull Request

- [x] Run formatting
- [x] Commented the code

## Changes description

add allContributors to recognise contributors to the project

## Comments

This will help to recognise and acknowledge all those who have contributed to the project.

### CODE REVIEW

**Comment by adrianvrj** [comment]
@raizo07 could you please resolve the conflicts to merge this?

**Comment by raizo07** [comment]
Done @adrianvrj You can merge now

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Go Stark Me is a modern platform designed to make fundraising for important caus
 
 Join us on Go Stark Me and make a difference by supporting meaningful projects in a secure and efficient manner.
 
+## UML Diagram
+
+![UML Class Diagram](https://github.com/user-attachments/assets/479c9296-e3ac-4ad3-bf79-5f458c456a45)
+
```
Comment (adrianvrj) [inline README.md:16]: this UML diagram should no longer be here, please remove

**Review by adrianvrj — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raizo07) [reply README.md]: resolved @adrianvrj

**Comment by raizo07** [comment]
@adrianvrj kindly review it

File: README.md [hunk 2]
```diff
@@ -31,10 +31,34 @@ If you are interested in contributing to Go Stark Me, please follow these steps:
...
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification.
+Contributions of any kind welcome!
+
 - [@bitfalt](https://t.me/bitfalt)
```
Comment (adrianvrj) [inline README.md:64]: This should be up there with the other TG handles

**Review by adrianvrj — CHANGES_REQUESTED** [review]

**Review by adrianvrj — APPROVED** [review]

**Comment by raizo07** [comment]
@all-contributors please add @raizo07 for code

**Comment by raizo07** [comment]
@all-contributors please add @raizo07 for code

**Pull request merged**